rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2047,6 +2047,22 @@ void getLastRetiredFederationP2SHScript_afterRSKIP186_whenSavingScript_returnsSc
         assertThat(lastRetiredFederationP2SHScript.get(), is(p2shScript));
     }
 
+    @Test
+    @Tag("clear retired federation")
```

Comment:
What's the benefit of using tags in this way with specific values for each test?  According to the [documentation](https://junit.org/junit5/docs/5.0.2/api/org/junit/jupiter/api/Tag.html) the intended use is for group common types of test.

Reply:
tbh it was inertia jaja, i can remove it

Reply:
If it is in more places(prs already merged), let it be as it is for now, and let's discuss this with the rest of team later.